org.eclipse.scout.rt.shared.test/src/test/java/org/eclipse/scout/rt/shared/services/lookup/CodeLookupCallTest.java

```diff
@@ -69,18 +69,24 @@ public void testGetDataByText() {
 
   @Test
   public void testGetDataByTextFiltered() {
+    // test text filters which yield the same result as the legacy call.
     runGetDataByTextFiltered(null);
     runGetDataByTextFiltered("*or*");
-    runGetDataByTextFiltered("*or");
```

Comment:
I'd prefer if you'd simply delete this test(line), instead of adding more complexity just for this odd legacy lookup call case. The rest looks Ok.